packages/components/src/DateTimePickers/DateTime/Manager/Manager.component.test.js

```diff
@@ -1,236 +1,245 @@
+					expectedDate: '2015aze-01-15',
 					expectedTime: undefined,
 				},
 				{
-					name: 'from props invalid date',
-					initialDate: new Date(),
-					newDate: new Date(''), // invalid date
-					expectedDate: undefined,
+					name: 'with empty string',
+					textInput: '',
+					expectedDate: '',
 					expectedTime: undefined,
 				},
 				{
-					name: 'from props valid date',
-					initialDate: new Date(),
-					newDate: new Date(2015, 3, 4, 12, 36),
-					expectedDate: new Date(2015, 3, 4),
-					expectedTime: { hours: '12', minutes: '36', seconds: '00' },
-				},
-				{
-					name: 'from props valid date with seconds',
-					initialDate: new Date(),
-					newDate: new Date(2015, 3, 4, 12, 36, 30),
-					expectedDate: new Date(2015, 3, 4),
-					expectedTime: { hours: '12', minutes: '36', seconds: '30' },
-					useSeconds: true,
+					name: 'with custom date format',
+					textInput: '15/01/2015',
+					expectedDate: new Date(2015, 0, 15),
+					expectedTime: undefined,
+					dateFormat: 'DD/MM/YYYY',
 				},
-			],
-		);
-
-		cases(
-			'props update should NOT update state',
-			({ initialDate, newDate }) => {
+			])('$name', async ({ expectedDate, expectedTime, textInput, dateFormat, useSeconds }) => {
 				// given
-				const wrapper = mount(
-					<Manager id={DEFAULT_ID} value={initialDate}>
-						<DateTimeConsumer />
+				const getProps = jest.fn();
+				render(
+					<Manager id={DEFAULT_ID} dateFormat={dateFormat} useSeconds={useSeconds}>
+						<DateTimeConsumer testDate={textInput} getProps={getProps} />
 					</Manager>,
 				);
-				const previousState = wrapper.find('DateTimeConsumerDiv').props();
 
 				// when
-				act(() => {
-					wrapper.setProps({
-						value: newDate,
-					});
-				});
-				wrapper.update();
-
+				await userEvent.click(screen.getByTestId('onDateChange'));
+				await userEvent.click(screen.getByTestId('getProps'));
 				// then
-				const nextState = wrapper.find('DateTimeConsumerDiv').props();
-				expect(previousState.date).toEqual(nextState.date);
-				expect(previousState.time).toEqual(nextState.time);
-			},
-			[
-				{
-					name: "when date ref doesn't change",
-					initialDate: undefined,
-					newDate: undefined,
-				},
-				{
-					name: 'when the dates are equals',
-					initialDate: new Date(2015, 1, 5, 21, 52),
-					newDate: new Date(2015, 1, 5, 21, 52),
-				},
-			],
-		);
-		describe('on change', () => {
-			cases(
-				'should update state when date change',
-				({ expectedDate, expectedTime, textInput, dateFormat, useSeconds }) => {
-					// given
-					const event = { target: { value: textInput } };
-					const wrapper = mount(
-						<Manager id={DEFAULT_ID} dateFormat={dateFormat} useSeconds={useSeconds}>
-							<DateTimeConsumer />
-						</Manager>,
-					);
-
-					// when
-					act(() => {
-						wrapper
-							.find('DateTimeConsumerDiv')
-							.props()
-							.onDateChange(event, { date: expectedDate, textInput, errors: [] });
```

Comment:
oh my god. calling onchange by hand with expectedDate and check the arg is expected :)